oneview/data_source_server_profile_template.go

```diff
@@ -170,5 +371,83 @@ func dataSourceServerProfileTemplateRead(d *schema.ResourceData, meta interface{
 		d.Set("boot_order", bootOrder)
 	}
 
+	overriddenSettings := make([]interface{}, 0, len(spt.Bios.OverriddenSettings))
+	for _, overriddenSetting := range spt.Bios.OverriddenSettings {
+		overriddenSettings = append(overriddenSettings, map[string]interface{}{
+			"id":    overriddenSetting.ID,
+			"value": overriddenSetting.Value,
+		})
+	}
+	if spt.Bios != nil {
+		biosOptions := make([]map[string]interface{}, 0, 1)
+		biosOptions = append(biosOptions, map[string]interface{}{
+			"manage_bios":         spt.Bios.ManageBios,
+			"overridden_settings": overriddenSettings,
+		})
+
+		d.Set("bios_option", biosOptions)
+	}
+
+	localStorages := make([]map[string]interface{}, 0, 1)
+	localStorages = append(localStorages, map[string]interface{}{
+		//"manage_local_storage": serverProfile.LocalStorage.ManageLocalStorage,
```

Comment:
any specific reason to why we are not capturing the manage_local_storage and initialize value, is not only required at time of creating the resource?

Reply:
manage local storage and initialize value is not part of the local storage.
http://apidocs.vse.rdlabs.hpecorp.net/fusion/2200/synergy/public/#rest/server-profiles

Reply:
> any specific reason to why we are not capturing the manage_local_storage and initialize value, is not only required at time of creating the resource?
Those fields are not part of local storage
http://apidocs.vse.rdlabs.hpecorp.net/fusion/2200/synergy/public/#rest/server-profiles